src/types/routerTypes.tsx

```diff
@@ -396,9 +396,8 @@ export function withNavigation<Props>(Component: React.ComponentType<Props>): Re
       push(name, params) {
         props.navigation.push(name, { route: { name, params } })
       },
-      replace(...a) {
-        // prev, next, params) {
-        props.navigation.replace(...a) // prev, next, next, { route: { params } })
+      replace(name, params) {
+        props.navigation.replace(name, { route: { name, params } })
```

Comment:
Yes, this is wrong. Jon accidentally broke the navigation shims, but we got that reverted, and now you just pass the params as the 2nd argument, not this structure.

Reply:
I don't understand what I need to fix here? My original changes to this file match what Jon made in commit 2c8b52e6587be59894e519bfaaf5f77a92a23f18

Reply:
It would be wrong because `navigation.replace` does the work, so doing it again here would double-nest the params. However, I see that this line gets eliminated anyhow, so it's moot.